compiler/src/iree/compiler/Codegen/LLVMGPU/test/transform_vector_to_mma.mlir

```diff
@@ -59,3 +59,166 @@ transform.sequence failures(propagate) {
   transform.iree.apply_patterns %func { canonicalization } : (!pdl.operation) -> ()
 }
 }
+
+// -----
+
+#map = affine_map<()[s0] -> (s0 * 32)>
+#map1 = affine_map<()[s0] -> (-s0 + 2)>
+#map2 = affine_map<()[s0] -> (s0 * -32 + 1024)>
+#map3 = affine_map<()[s0] -> (s0 * -32 + 64)>
+#map4 = affine_map<()[s0] -> (s0 mod 8)>
+#map5 = affine_map<()[s0, s1] -> (((s0 + s1 * 8) mod 32) floordiv 8)>
+#map6 = affine_map<()[s0] -> (s0 * 4 - (s0 floordiv 8) * 32)>
+#map7 = affine_map<()[s0, s1] -> ((((s0 + s1 * 8) mod 32) floordiv 8) * 4)>
+#map8 = affine_map<()[s0] -> (s0 * -4 + (s0 floordiv 8) * 32 + 32)>
+#map9 = affine_map<()[s0, s1] -> ((((s0 + s1 * 8) mod 32) floordiv 8) * -4 + 16)>
+#map10 = affine_map<()[s0] -> (-s0 + 64)>
+#map11 = affine_map<()[s0] -> (-s0 + 576)>
+#map12 = affine_map<(d0, d1, d2) -> (d0, d2)>
+#map13 = affine_map<(d0, d1, d2) -> (d2, d1)>
+#map14 = affine_map<(d0, d1, d2) -> (d0, d1)>
+hal.executable private @convolution  {
+builtin.module {
+// CHECK-LABEL: func.func @convolution
+func.func @convolution() {
```

Comment:
please reduce the test and try to only test one thing.

Reply:
I isolated the rank 1 elementwise + broadcast from the example which was causing crashes. I kept it with checks for subgroup mma ops because I want to make sure that unroll + conversion to mma still works with this present. If wanted I can merge this test with the previous one though because much of it is duplicated.